File: app/src/main/java/com/shahu/weathery/model/forecast/ForecastResponse.java
package com.shahu.weathery.model.forecast;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("com.robohorse.robopojogenerator")
public class ForecastResponse {

	@SerializedName("city")
	private City city;

	@SerializedName("cnt")
	private int cnt;

	@SerializedName("cod")
	private String cod;

	@SerializedName("message")
	private int message;

	@SerializedName("list")
	private List<ForecastListItem> list;

	public void setCity(City city){
		this.city = city;
	}

	public City getCity(){
		return city;
	}

	public void setCnt(int cnt){
		this.cnt = cnt;
	}

	public int getCnt(){
		return cnt;
	}

	public void setCod(String cod){
		this.cod = cod;
	}

	public String getCod(){
		return cod;
	}

	public void setMessage(int message){
		this.message = message;
	}

	public int getMessage(){
		return message;
	}

	public void setList(List<ForecastListItem> list){
		this.list = list;
	}

	public List<ForecastListItem> getList(){
		return list;
	}

	@Override
 	public String toString(){
		return 
			"ForecastResponse{" +
			"city = '" + city + '\'' + 
			",cnt = '" + cnt + '\'' + 
			",cod = '" + cod + '\'' + 
			",message = '" + message + '\'' + 
			",list = '" + list + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/shahu/weathery/model/forecast/ForecastResponse.kt
package com.shahu.weathery.model.forecast

import com.google.gson.annotations.SerializedName
import javax.annotation.Generated

@Generated("com.robohorse.robopojogenerator")
class ForecastResponse {
    @SerializedName("city")
    var city: City? = null

    @SerializedName("cnt")
    var cnt = 0

    @SerializedName("cod")
    var cod: String? = null

    @SerializedName("message")
    var message = 0

    @SerializedName("list")
    var list: List<ForecastListItem>? = null

    override fun toString(): String {
        return "ForecastResponse{" +
                "city = '" + city + '\'' +
                ",cnt = '" + cnt + '\'' +
                ",cod = '" + cod + '\'' +
                ",message = '" + message + '\'' +
                ",list = '" + list + '\'' +
                "}"
    }
}